package/system-role-server-default.spec

```
@@ -66,15 +66,16 @@ mkdir -p $RPM_BUILD_ROOT
 #
 # Add control file
 #
-install -m 644 control/installation.xml $RPM_BUILD_ROOT/
+mkdir -p $RPM_BUILD_ROOT/%{_datadir}/system-roles
+install -m 644 control/installation.xml $RPM_BUILD_ROOT/%{_datadir}/system-roles/server-default.xml
```

I'm just thinking how to avoid file conflict when using copy&paste for this line.

Something like 

```suggestion
install -m 644 control/installation.xml $RPM_BUILD_ROOT/%{_datadir}/system-roles/%{role_name}.xml
```

would be better together with

```
%define role_name server-default
Name:           system-role-%{role_name}
```

(But I'm actually not sure if a macro is allowed in the package name, please test...)